docs/content/preview/admin/yb-ts-cli.md

```diff
@@ -273,6 +273,14 @@ yb-ts-cli [ --server_address=<host>:<port> ] refresh_flags
 
 * *host*:*port*: The *host* and *port* of the YB-Master or YB-TServer. Default is `localhost:9100`.
 
+##### Clear the current metacache on a tablet server or master server
+**Syntax**
```

Comment:
Please add a description and use the same format as the other commands. Explain if it works on both yb-tserver/master, and 1-2 sentence what metacache is.

Reply:
Ok, Will do